stories/splash.stories.js

```diff
@@ -76,7 +80,7 @@ storiesOf( 'Splash', module )
 							mdc-layout-grid__cell--span-3-desktop
 						">
 							<DashboardSplashModule
-								icon="pagespeed"
+								icon={ <PageSpeedInsightsIcon height="33" /> }
```

Comment:
This is missing the `width` I assume?

Reply:
This is a leftover before the consistency fix was implemented, good catch.